npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb

```diff
@@ -1174,6 +1174,92 @@
         end
       end
 
+      context "specified using 'packages/*/*'" do
+        before do
+          stub_request(:get, File.join(url, "package.json?ref=sha")).
+            with(headers: { "Authorization" => "token token" }).
+            to_return(
+              status: 200,
+              body:
+                fixture("github", "package_json_with_nested_glob_workspaces.json"),
+              headers: json_header
+            )
+          stub_request(
+            :get,
+            File.join(url, "packages/package1?ref=sha")
+          ).with(headers: { "Authorization" => "token token" }).
+            to_return(
+              status: 200,
+              body: fixture("github", "packages_files_nested2.json"),
+              headers: json_header
+            )
+          stub_request(
+            :get,
+            File.join(url, "packages/package2?ref=sha")
+          ).with(headers: { "Authorization" => "token token" }).
+            to_return(
+              status: 200,
+              body: fixture("github", "packages_files_nested3.json"),
+              headers: json_header
+            )
+          stub_request(
+            :get,
+            File.join(url, "packages/package1/package1/package.json?ref=sha")
+          ).with(headers: { "Authorization" => "token token" }).
+            to_return(
+              status: 200,
+              body: fixture("github", "package_json_content.json"),
```

Comment:
Ah I realized `package_json_content.json` also base64 encodes its content.
https://github.com/dependabot/dependabot-core/blob/main/npm_and_yarn/spec/fixtures/github/package_json_content.json

Maybe we should also include a comment with the plaintext decoded content for that?

```json
{
  "name": "bump-test",
  "version": "0.0.1",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/gocardless/bump-test.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/gocardless/bump-test/issues"
  },
  "homepage": "https://github.com/gocardless/bump-test#readme",
  "dependencies": {
    "lodash": "^1.3.1",
    "chalk": "0.4.0",
    "stopwords": "0.0.1"
  },
  "devDependencies": {
    "etag": "^1.0.0"
  }
}
```